release_notes/ocp-4-1-release-notes.adoc

```diff
@@ -886,7 +886,39 @@ update could not be applied.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1711964[*BZ#1711964*])
 
 * The installer may have a `0` return code when hitting a FATAL event.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1712409[*BZ#1712409*])
+
+* After disabling service catalog, attempting to delete projects on the cluster
+can get stuck in a "Terminating" state. This is caused by the service catalog
+API service not deleting itself properly, and the deletion hangs. Service
+catalog does not properly clean up service instance resources that have a
+`kubernetes-incubator/service-catalog` finalizer; as a result, the namespace
+controller cannot finish the namespace (project) deletion.
++
+A change was made to service catalog to prevent the hangs by no longer
+attempting to delete its API service after disabling service catalog. If you
+have any projects stuck in a "Terminating" state after deleting service catalog,
+cluster administrators can use the following workaround:
++
+--
+. If you want to retain existing services that were deployed using service
+catalog, you must manually remove any `ownerReferences` fields in the secrets
+used by related service binding resources. Failure to do so results in the
+secrets being deleted along with the service bindings when the API service is
+removed, and applications may no longer be able to access the service.
++
+NOTE: This step can be skipped if you do not want to retain the existing services.
+
+. Manually delete the API service:
++
+----
+$ oc delete apiservice v1beta1.servicecatalog.k8s.io
+----
+--
++
+After implementing this workaround, you can continue deleting projects without
+the service catalog causing an issue.
```

Comment:
Not critical, but the wording "without the service catalog causing an issue" sounds just a little funny. Could you maybe just say "without issue"?